.travis.yml

```diff
@@ -2,14 +2,16 @@ language: node_js
 sudo: required
 node_js:
   - "0.10"
+  - "4"
   - "4.4.3"
+  - "6"
```

Comment:
We should be testing against recent versions of Node.js. I kept 4.4.3 since I believe we have some requirements around supporting that explicit version

Reply:
Good spot! Let's remove 0.10 - it's gone for long time.